src/components/books.js

```diff
@@ -1,25 +1,26 @@
-import React, { useState } from 'react';
-import Book from './book';
-import AddBook from './AddBook';
+/* eslint-disable import/extensions */
+/* eslint-disable react/prop-types */
```

Comment:
Linter checks help to maintain a coding standard between you and your teammates and most of the time suggest best coding practices. As much as possible, kindly avoid modifying them.